### DISCUSSION TITLE
Toggle Window to Whole Screen and Go Back

### DISCUSSION THREAD

**Discussion by jrock2004** [asmvik/yabai#2346] [Q&A] [answered]
So I have a 2 column layout, but when I am on my laptop I want to temp make the window I am working in take the whole screen. Wondering the command I need to do this?

```
-------------------
|        |        |
|        |        |
|    A   |   B    |
|        |        |
|        |        |
-------------------
```
hoping to key bind a yabai command to make window A take up the whole screen. Then when I hit the command again it goes back to above. Is this possible?

**Comment by asmvik** [comment, marked as answer]
yabai -m window --toggle zoom-fullscreen

**Reply by jrock2004** [reply]
Thank you
